packages/block-directory/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Breaking Change
+
+-   Upgraded React components to work with v17.0 ([#29118](https://github.com/WordPress/gutenberg/pull/29118)). There is no new features in React v17.0 as explained in the [blog post](https://reactjs.org/blog/2020/10/20/react-v17.html).
```

Comment:
`There is no` -> `There are no`?

Reply:
Good catch. I hear both forms in a spoken language from native speakers but it's probably `there are` that is correct in written form.